cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -115,6 +115,13 @@
                             block__flush-top">
                     {{ render_stream_child(block) }}
                 </div>
+            {% elif 'post_preview_snapshot' in block.block_type %}
```

Comment:
bug scenario:
If the sublanding page has the post_preview_snapshot as the first child in the stream field it never hits this conditional. suggested change.
```
{% if loop.index == 1 %}
                <div class="block
                            block__flush-top">
            {% else %}
                <div class="block">
            {% endif %}

            {% if 'post_preview_snapshot' in block.block_type %}
                {% set posts = page.get_browsefilterable_posts(request) %}
                    {% for i in range(block.value.limit | int) %}
                        {{ post_preview.render(posts[i], '', 0, pageurl(posts[i].parent())) }}
                    {% endfor %}
                </div>
            {% else %}
                    {{ render_stream_child(block) }}
                </div>
            {% endif %}
```